.github/ISSUE_TEMPLATE/experimental_feature.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Experimental Feature Requirements
+about: Requirements for experimental features
+---
+
+This page lists the requirements for experimental features. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+
+--- 
+
+### Requirements: 
+
+- [ ] Feature is reviewed in a work group meeting
+- [ ] RFC has been written
+- [ ] Feature is disabled by default
+- [ ] Steps for enabling the feature have been documented. This may include
+	instructions for building, running an `istioctl experimental command`, or
+	using the preview profile.
+```
+Link to instructions
+```
+- [ ] Plan exists for getting feedback. This may include user feedback meetings,
+	discuss.istio.io conversations, GitHub issues, or mailing lists. 
+```
+Plan for feedback
+```
+- [ ] No impact on performance when the feature is disabled.
```

Comment:
I'd move this up to be right after the one about disabled by default, ties the two together. It is off by default and it doesn't hurt anything when it is off.